tests/apps/pubsub-publisher/app.go

```diff
@@ -85,12 +86,16 @@ func performPublish(w http.ResponseWriter, r *http.Request) {
 		})
 		return
 	}
+	if len(commandBody.PubSubName) == 0 {
+		commandBody.PubSubName = pubsubName
+	}
 
 	log.Printf("    commandBody.ContentType=%s", commandBody.ContentType)
 	log.Printf("    commandBody.Topic=%s", commandBody.Topic)
 	log.Printf("    commandBody.Data=%v", commandBody.Data)
 	log.Printf("    commandBody.Protocol=%s", commandBody.Protocol)
 	log.Printf("    commandBody.Metadata=%s", commandBody.Metadata)
+	log.Printf("    commandBody.PubSubName=%s", commandBody.PubSubName)
```

Comment:
Please, do not make this test aware of pubsub. We keep the test generic.

Reply:
yes, I agree why this is preferable. Maybe we can run the test for all supported brokers and make it somehow configurable, as not all brokers support the same features.